src/qml/README.md

```diff
@@ -71,3 +71,9 @@ you must configure with the following option:
 ```
 ./configure --with-qml
 ```
+### Run
+
+To run the qml GUI, run the following command from the root directory.
```

Comment:
I think the current text isn't optimal because:
1) Generally throughout our documentation, commands are written assuming that you are executing them from the root of the cloned repo. Including the text "... run the following command from the root directory" is redundant information, and actually a bit confusing (system root or repo root?)
2) The text has an awkward flow to it. We are saying "To run A, run B"

this really could just be the following, no more information is needed:
```suggestion
To run the qml GUI:
```